src/GitVersionCore/Core/Abstractions/Git/IRemote.cs

```diff
@@ -0,0 +1,8 @@
+namespace GitVersion
+{
+    public interface IRemote
```

Comment:
```suggestion
    public interface IRemote : IEquatable<IRemote>, IComparable<IRemote>
```

Reply:
Agree, I will change that in a subsequent PR